docs/faststore/docs/go-live/2-configuring-external-dns.mdx

```diff
@@ -48,9 +48,11 @@ Now, you must set up your VTEX account to use the DNS records created in the pre
 4. In the **Hosts** field, enter your main domain (e.g., `mystore.com`) to the list.
 5. Click `Add`.
 6. Add the `secure` subdomain (e.g., `secure.mystore.com`) to the list.
-   ![Host configuration](https://vtexhelp.vtexassets.com/assets/docs/src/hostsConfig___cad00ec89023cd77c34f77898cda1b3e.png)
+   ![host-configuration](https://vtexhelp.vtexassets.com/assets/docs/src/account-settings-new___2e13bee3041e8505922a737b94652ca6.png)
 7. Click the **Save** button.
 
+> ⚠️ To ensure the correct functioning of the checkout for all customer access scenarios, all subdomains used by the store must be added to the host list. If the store operates without the `www.` domain, you must include both the `www.` and non-`www.` domains in the Hosts list. This guarantees that the checkout process works seamlessly when customers access the store with or without `www.`.
```

Comment:
Pode referenciar essa doc: https://help.vtex.com/en/tutorial/configuring-the-store-domain--tutorials_2450
Seria bom validar se realmente tem que colocar a rota sem www. Se for o caso, talvez valha a pena atualizar a doc https://help.vtex.com/en/tutorial/configuring-the-store-domain--tutorials_2450 , falando de exceções no caso da FastStore.


c.c.: @julia-rabello

Reply:
Boa! 
No caso de colocar a rota se www, veio de um problema que Ícaro, Emerson e Bibi passaram com um cliente ([aqui](https://vtex.slack.com/archives/C03L3CRCDC4/p1709155931177009?thread_ts=1709154031.055469&cid=C03L3CRCDC4)) e precisava estar mais claro na doc. Parte da conversa:
<img width="740" alt="Screenshot_458" src="https://github.com/vtexdocs/dev-portal-content/assets/67270558/ea47c21e-af04-416f-8329-84231168bd0d">
<img width="728" alt="Screenshot_459" src="https://github.com/vtexdocs/dev-portal-content/assets/67270558/f07f2154-f0e3-47cc-bce8-7e3e1ba2c02b">